.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
     - TASK=ci-unit NODE_INDEX=0 NODE_TOTAL=2
     - TASK=ci-unit NODE_INDEX=1 NODE_TOTAL=2
     - TASK=ci-integration
+    - TASK=ci-py3-tests
```

Comment:
Can we name it more explicitly here to make clear that `st2client` tests are executed under `py3`?